chore: update d2 cluster guide (#20)
dhis2/developer-portal: docs/guides/spin-up-local-instance.md

password: district
```

### Destroy and cleanup commands    

The `down` command followed by the name of the cluster shuts down all Docker processes connected to that cluster (it basically destroys a running container):  

```shell
d2 cluster down 2.35.0
```

> The command `down --clean` destroys that cluster _and_ clears out all data volumes as well as ephemeral containers: 

Reviewer: ```suggestion
The command `down --clean` destroys that cluster _and also_ clears out all saved configuration, data volumes, and ephemeral containers: 
```

Reviewer: Looks good!  Technically it also clears out the saved configuration for this cluster from the local filesystem
